content/blog/reflections-of-a-pulumi-intern/index.md

```diff
@@ -0,0 +1,66 @@
+---
+title: "Reflections of a Pulumi Intern"
+meta_image: meta.png
+authors: ["albert-zhong"]
+meta_desc: "A reflection of my Pulumi intern experiences"
+tags: ["pulumi", "intern"]
+date: "2020-09-18"
+---
+
+Hi everyone! I'm Albert, a soon-to-be sophomore studying computer science at the University of Washington. Today marks my last day as a Pulumi intern, so I figure I'd reflect on my experiences up until this point.
+
+## Joining Pulumi
+
+I heard of Pulumi for the first time when they visited my school's career fair in January. As I rounded the corner into the [CSE1 atrium](https://www.cs.washington.edu/building), I saw a banner titled "Modern Infrastructure as Code," with some lines of TypeScript of what appeared to be the creation of an S3 bucket. I found it interesting because it looked so simple and modular. In contrast, when I had played around with AWS in the past to deploy the ocassional Django web app, it had always involved following a series of exact instructions (usually go to this dashboard, copy this YAML file, run this command, and pray it works).
+
+I walked up to the banner's adjacent booth and chatted with [Sean](https://www.pulumi.com/blog/author/sean-holung/), a software engineer at Pulumi. He explained how Pulumi was building an open-source platform that allows developers to manage cloud infrastructure using familiar programming languages. To be honest I didn't have too much experience with the cloud, but I did sympathize with the pain of wrangling with YAML files. The concept sounded compelling, so I told him I was interested in interning there. He gave me the Pulumi recruiter's email, who I later emailed that day with my resume and a brief bit about my background. After a few email and phone exchanges, I came over to the Pulumi office for an in-person interview (which was unforunately also the only time I ever saw it, due to [COVID-19](https://www.pulumi.com/blog/coronavirus-plan/)).
```

Comment:
The diff doesn't clearly show the changes, but it's just updating the two links:

 - `[Sean](/blog/author/sean-holung/)`
 - `[COVID-19]({{< relref "/blog/coronavirus-plan" >}})`

`relref` ensures the page exists when Hugo builds the site. Would love to be able to use it for the author pages, but it doesn't work for those, so just use relative links on their own.


```suggestion
I walked up to the banner's adjacent booth and chatted with [Sean](/blog/author/sean-holung/), a software engineer at Pulumi. He explained how Pulumi was building an open-source platform that allows developers to manage cloud infrastructure using familiar programming languages. To be honest I didn't have too much experience with the cloud, but I did sympathize with the pain of wrangling with YAML files. The concept sounded compelling, so I told him I was interested in interning there. He gave me the Pulumi recruiter's email, who I later emailed that day with my resume and a brief bit about my background. After a few email and phone exchanges, I came over to the Pulumi office for an in-person interview (which was unforunately also the only time I ever saw it, due to [COVID-19]({{< relref "/blog/coronavirus-plan" >}})).
```